cac-test-local.yml

```diff
@@ -93,7 +93,9 @@ jenkins:
             value: -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
           - key: GRADLE_OPTS
             value: -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
+  disabledAdministrativeMonitors:
```

Comment:
new feature in 2.230, which i've just been manually doing to get rid of the warnings we don't care about